src/processor/Package.ts

```diff
@@ -228,4 +230,44 @@ export class Package {
       pullAt(sd.rules, rulesToRemove);
     });
   }
+
+  private suppressUrlAssignmentOnExtensions(): void {
+    // Loop over all profiles and extensions, removing assignment rules on inline extensions
+    [...this.profiles, ...this.extensions].forEach(sd => {
+      const rulesToRemove: number[] = [];
+      sd.rules.forEach(rule => {
+        if (rule instanceof ExportableContainsRule && rule.path.endsWith('extension')) {
```

Comment:
I think this applies to `modifierExtension` as well, so the test should probably be:
```ts
/(modifierE|e)xtension$/.test(rule.path)
```